src/http/plugins/db.ts

```diff
@@ -209,6 +213,7 @@ export const migrations = fastifyPlugin(
         const tenant = await getTenantConfig(request.tenantId)
         const migrationsUpToDate = await areMigrationsUpToDate(request.tenantId)
 
+        console.log('MIGRATIONS UP TO DATE', migrationsUpToDate)
```

Comment:
Is this an log intentional or left over debugging?

Reply:
left over 👍